### DISCUSSION TITLE
helper functions for [String] and [Text]

### DISCUSSION THREAD

**Discussion by clojj** [chris-martin/loc#10] [General]
as this packages types are often used in conjunction with [Text] or [String]
I think it would make sense to include some helper functions here.

for exampe:
Span -> [Text] -> [Text]
or
Span -> [Text] -> Text

I can put these in a PR if you want ?

**Comment by chris-martin** [comment]
Yes, I think this is a natural request. If you have any code to suggest, I'd be happy to see it. If I recall correctly, I didn't include anything like this because I had some difficulty reasoning about some corner cases at line breaks.
